### DISCUSSION TITLE
Nginx Lua Agent

### DISCUSSION THREAD

**Discussion by trs91km** [apache/skywalking#8197] [Q&A]
With skywalking nginx lua agent , can we get nginx metrics for nginx connection counts like active, waiting, reading, writing?
in Skywalking showcase, there is demo only for java agent, is there any demo setup available for nginx lua agent?

**Comment by wu-sheng** [comment]
You don't need agent to do so. You could use OpenTelemetry collector to collect metrics through Prometheus endpoint, then forward to SkyWalking OAP server.

@dmsolr Do you have more to share?

**Reply by trs91km** [reply]
Similar setup we can use to get varnish metrics as well?
